asreview/webapp/authentication/oauth_handler.py

```diff
@@ -0,0 +1,154 @@
+import requests
+
+
+class OAuthHandler:
```

Comment:
Maybe this class could use a dedicated exception like `OAuthException`. That way when this class is called upon somewhere in the code, we know which exception we get if something happens in the oauth process.